libompd/src/omp-icv.cpp

```diff
@@ -11,7 +11,10 @@
     macro (debug_var, "debug-var", ompd_scope_address_space, 0)                \
     macro (nthreads_var, "nthreads-var", ompd_scope_thread, 0)                 \
     macro (display_affinity_var, "display-affinity-var", ompd_scope_address_space, 0)    \
+    macro (affinity_format_var, "affinity-format-var", ompd_scope_address_space, 0)      \
```

Comment:
Could you explain why each of these three ICVs are needed? I'm not saying it's wrong, I just don't understand their usefulness to the debugger. Simply to display to the user?

Reply:
A simple answer might be: because the OpenMP spec requires the OMPD library to expose all ICV values.

In the tool, I wouldn't display all those ICV by default, but rather on user request. So, probably somehow list those ICV as "available in this context" as soon as OMPD is loaded and you queried the list of available ICV names.

By adding such ICV to the observed variable list, the user could select the ICV of interest. 
Probably make them also available for conditional breakpoints?

Reply:
Partially, since, I had a 'info omp-icv [optional icv name]' gdb command in mind for doing this (where the ICV names and values are listed) and partially, to address the LLVM-OMPD implementation gaps wrt what @jprotze mentioned.

Reply:
Thanks, that makes sense.